hugo/assets/ts/helpers/cookie-service.ts

```diff
@@ -1,15 +1,23 @@
+export enum CookieCategory {
+    NECESSARY = 'necessary',
+    PREFERENCES = 'preferences',
+    STATISTICS = 'statistics',
+    MARKETING = 'marketing'
+}
+
 export class CookieService {
-    static hasAcceptedCookies(): boolean {
-        return (CookieService.getCookie('cue-cookies') === 'yes');
+    static hasConsent(_category: CookieCategory): boolean {
```

Comment:
I think we might have to update the linter config to ignore this pattern (_argument).

```
"@typescript-eslint/no-unused-vars": [
          "warn",
          {
            "argsIgnorePattern": "^_"
          }
        ],